README.md

```diff
@@ -1,9 +1,9 @@
 # Sudan-bot
 ## Installation
-1. clone this repo
-2. in bot/ rename `Config.sample` to `Config`
-3. again in bot/ edit `Config/config.ini` and `Config/firebase.json`
-4. install requirements from `requirements.txt` this can be done by passing the `-r` flag to pip followed by the requirements file if for some reason you want to use `start.py` you will also have to install `termcolor`
-5. `cd bot` and run `core.py` or you can run `start.py` if you want a bit of a ui but its a bit useless if you are on linux and want to use the auto updating cog you will have to run it with `start.sh`
+1. Clone this repo.
+2. In `bot/` rename `Config.sample` to `Config`.
+3. Again in `bot/` edit `Config/config.ini` and `Config/firebase.json`.
+4. Install requirements from `requirements.txt`. This can be done by passing the `-r` flag to pip followed by the requirements file.
+5. `cd bot` and run `core.py`. If you are on linux and want to use the auto updating cog you will have to run it with `start.sh`.
 
-For more info look at the [Wiki](https://github.com/TheSuperGamer20578/Sudan-bot/wiki)
+For more info look at the (currently non-existant) [Wiki](https://github.com/TheSuperGamer20578/Sudan-bot/wiki)

```

Comment:
@ogxn is this what you mean in your review?

Reply:
yes but please note this is literally ONE file

Reply:
yes but there are NINE lines it could be any one